### PR TITLE
String value pattern updated in PrimitiveJsonPropertyBodyFilter (StackOverflowError fix)

### PR DESCRIPTION
**Pull request by dicody**
Support huge json values filtration (StackOverflowError fix)

## Description
Regexp string value pattern updated

## Motivation and Context
https://github.com/zalando/logbook/issues/1327

## Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.

### CODE REVIEW

File: logbook-json/src/test/resources/huge-json-value.json [hunk 1]
```diff
@@ -1,4 +1 @@
...
...tityPersister\",\"nativeMethod\":false},{\"methodName\":\"check\",\"fileName\":\"AbstractEntityPersister.java\",\"lineNumber\":2479,\"className\":\"org.hibernate.persister.entity.AbstractEntityPersister\",\"nativeMethod\":false},{\"methodName\":\"check\",\"fileName\":\"AbstractEntityPersister.java\",\"lineNumber\":2479,\"className\":\"org.hibernate.persister.entity.AbstractEntityPersister\",\"nativeMethod\":false},{\"methodName\":\"check\",\"fileName\":\"AbstractEntityPersister.java\",\"lineNumber\":2479,\"className\":\"org.hibernate.persister.entity.AbstractEntityPersister\",\"nativeMethod\":false},{\"methodName\":\"check\",\"fileName\":\"AbstractEntityPersister.java\",\"lineNumber\":2479,\"className\":\"org.hibernate.persister.entity.AbstractEntityPersister\",\"nativeMethod\":false},{\"methodName\":\"check\",\"fileName\":\"AbstractEntityPersister.java\",\"lineNumber\":2479,\"className\":\"org.hibernate.persister.entity.AbstractEntityPersister\",\"nativeMethod\":false},{\"methodName\":\"check\",\"fileName\":\"AbstractEntityPersister.java\",\"lineNumber\":2479,\"className\":\"org.hibernate.persister.entity.AbstractEntityPersister\",\"nativeMethod\":false},{\"methodName\":\"check\",
```
[the remark this answers is not in the document]
Comment (dicody) [reply logbook-json/src/test/resources/huge-json-value.json]: thanks, updated the formatting

**Review by Klapsa2503 — APPROVED** [review]

**Comment by Klapsa2503** [comment]
Any chances to get it into the main branch? We had similar problem with JSON filter and testing proved that this fix works

**Comment by DaniloVeljovic** [comment]
👍🏼

**Comment by DaniloVeljovic** [comment]
@dicody Can you please pull the main branch into your branch, so that we can retrigger the builds? I can't merge this code because the builds were not triggered

**Comment by dicody** [comment]
> @dicody Can you please pull the main branch into your branch, so that we can retrigger the builds? I can't merge this code because the builds were not triggered

done

**Pull request merged**